### PR TITLE
Integrated Twilio Client and respective tests

### PR DESCRIPTION
**Pull request by liunick**

### CODE REVIEW

File: tests/twilio_test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Read config file
+config = configparser.ConfigParser()
+config.read('../config/config.ini')
+
+# Initialize the Twilio Client
+twilioClient = TwilioClient(config=config)
+
+# Goes through each phone # and runs the send message function if violation
+for i in range(phone_number.__len__()):
```
Comment (francesar) [inline tests/twilio_test.py]: `len(phone_number)`

File: tests/twilio_test.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import configparser
+import sys
+sys.path.append('../')
```
Comment (francesar) [inline tests/twilio_test.py]: whats this for

Comment (liunick) [reply tests/twilio_test.py]: importing a file from a different directory from the same parent dir

Comment (francesar) [reply tests/twilio_test.py]: are you trying to import `from twilio_client.twilio_client import TwilioClient`

**Pull request merged**